### PR TITLE
Append share-kit-from query param when updating the request eleme…

### PR DESCRIPTION
**Pull request by JaceHensley**
## Ultimate Problem
The share-kit-from query param was only being applied on the first render

## Solution
- Move the `appendQuery` to each individual `render` function so it gets called on first render and updates

### CODE REVIEW

File: src/elements/renderRequestQRCode.ts [hunk 1]
```diff
@@ -320,6 +322,7 @@ const renderRequestQRCode = (config: {
   requestData: RequestData
   qrOptions?: Partial<QROptions>
 }): RequestElementResult => {
+  console.log({requestData: config.requestData})
```
Comment (edhedges) [inline src/elements/renderRequestQRCode.ts]: Leaving this in?

Comment (JaceHensley) [reply src/elements/renderRequestQRCode.ts]: good catch!

File: test/renderRequestElement.test.ts [hunk 2]
```diff
@@ -90,64 +88,4 @@ describe('renderRequestElement', () => {
...
-        expect(btnSpy).toHaveBeenCalledWith(
-          expect.objectContaining({
-            requestData: expect.objectContaining({
-              url: 'https://receive-kit.bloom.co/api/receive?test=&share-kit-from=button',
-            }),
-          }),
-        )
-      })
-    })
-  })
```
Comment (edhedges) [inline test/renderRequestElement.test.ts]: Why remove these tests? No longer relevant?

Comment (JaceHensley) [reply test/renderRequestElement.test.ts]: Yeah renderRequestElement is no longer appending the query param

**Pull request merged**